Translations/hr.po

```diff
@@ -6088,27 +6634,97 @@ msgstr "čarolije"
 msgid "mute"
 msgstr "utišaj"
 
+#: Source/panels/spell_book.cpp:153 Source/panels/spell_list.cpp:162
+msgid "Skill"
+msgstr "Vještina"
+
+#: Source/panels/spell_book.cpp:157
+msgid "Staff ({:d} charge)"
+msgid_plural "Staff ({:d} charges)"
+msgstr[0] "Štap ({:d} napunjen)"
+msgstr[1] "Štap ({:d} napunjena)"
+msgstr[2] "Štap ({:d} napunjeno)"
+
+#. TRANSLATORS: UI constrains, keep short please.
+#: Source/panels/spell_book.cpp:167
+#, fuzzy
```

Comment:
Don't forget that translations marked as "needs work" or fuzzy will no appear in game.